pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -391,10 +411,51 @@ func (m *Manager) getKeyspaceNameByID(ctx context.Context, id uint32) (string, e
 		return "", fmt.Errorf("got an empty keyspace name by id %d", id)
 	}
 	// Update the cache.
-	m.keyspaceNameLookup[id] = loadedName
+	m.updateKeyspaceNameLookup(id, loadedName)
```

Comment:
Is it possible to change the keyspace name? If keyspace is modified or deleted, do we need to update it?

Reply:
Currently, once a keyspace is created, its ID and name cannot be changed. Therefore, we don’t need to consider cache misses at this stage.

Reply:
Currently, once a keyspace is created, its ID and name cannot be changed. Therefore, we don’t need to consider cache misses at this stage.